sse2neon.h

```diff
@@ -4591,9 +4591,17 @@ FORCE_INLINE void _mm_sfence(void)
     __sync_synchronize();
 }
 
+// Store 128-bits (composed of 4 packed single-precision (32-bit) floating-
+// point elements) from a into memory using a non-temporal memory hint.
```

Comment:
I wonder why the `mem_addr must be aligned on a 16-byte boundary` related description are ignored?

Reply:
My intention was to extract the first sentence from Intel/Microsoft online document. And no further sentences should be extracted for consistency.

Reply:
Not sure which one is better.
As I check the `sse2neon.h`, it seems like most of the comments of intrinsics are following the rule like extract the first sentence, and few of them are more than one sentence.